impeller/entity/contents/content_context.h

```diff
@@ -713,13 +713,15 @@ class ContentContext {
 
     void CreateDefault(const Context& context,
                        const ContentContextOptions& options,
-                       const std::initializer_list<int32_t>& constants = {}) {
+                       const std::initializer_list<int32_t>& constants = {},
+                       bool has_subpass_input = false) {
```

Comment:
Consider using an enum instead of a bool so that this reads more nicely at the callsites, e.g. `enum class UseSubpassInput { kYes, kNo, };`

Reply:
I think this is why object-c has YES and NO constants 😄